moulinette/utils/filesystem.py

```diff
@@ -77,6 +80,28 @@ def read_yaml(file_path):
     return loaded_yaml
 
 
+def read_toml(file_path):
+    """
+    Safely read a toml file
+
+    Keyword argument:
+        file_path -- Path to the toml file
+    """
+
+    # Read file
+    file_content = read_file(file_path)
+
+    # Try to load toml to check if it's syntaxically correct
+    try:
+        loaded_toml = toml.loads(file_content, _dict=OrderedDict)
+    except Exception as e:
+        raise MoulinetteError(errno.EINVAL,
+                              m18n.g('corrupted_toml',
```

Comment:
We're missing the `corrupted_toml` in the locales files?

Reply:
Ah yes indeed '^'